mvc/model.js

```diff
@@ -1,8 +1,10 @@
 const db = require("../db/connection");
 
 exports.readAllTopics = () => {
-  return db.query("SELECT * FROM topics").then((response) => {
-    return response.rows;
+  return db.query("SELECT * FROM topics").then(({ rows }) => {
+    if (rows === undefined) {
+      return Promise.reject({ status: 404, msg: "path not found" });
```

Comment:
You don't need this here and the functionality doesn't match the method. 

This error would be if no topics were returned from the database - not if a user has typed an invalid path